### ISSUE TITLE
Moving Thermometer on and off Temperature map causes scale to shrink

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/number-line-integers#81]
**Test device**
Windows 7 Laptop
**Operating System**
Windows 7
**Browser**
Chrome
**Problem description**
When thermometer is moved onto the temperature map the scale to the left may randomly shrink and return to normal
**Steps to reproduce**
1. Move Thermometer onto Temperature map
2. Move thermometer off and onto map until shrinking occurs
**Visuals**
![2020-01-14TempSclShrnkNoLn](https://user-images.githubusercontent.com/54685448/72387007-c6415a00-36df-11ea-84a5-d4d98b778645.gif)

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Number Line: Integers‬
URL: https://phet-dev.colorado.edu/html/number-line-integers/1.0.0-rc.1/phet/number-line-integers_en_phet.html
Version: 1.0.0-rc.1 2020-01-06 19:43:39 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.117 Safari/537.36
Language: en-US
Window: 1366x625
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 32 varying: 32 uniform: 256
Texture: size: 8192 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by KatieWoe** [comment]
For https://github.com/phetsims/QA/issues/465

**Comment by KatieWoe** [comment]
Seeing this on the new Chromium version of Edge

**Comment by jbphet** [comment]
There is one particular spot where the reduction in size of the panel with the number line occurs, and it is where the first value encountered is 32 degrees F, which is 0 degrees Celsius.   Here is a screenshot where the situation has been created and doesn't require any movement (note the relatively small number line panel):

![image](https://user-images.githubusercontent.com/3082831/72655463-f728c580-3951-11ea-9d97-49af200a1eed.png)

**Comment by jbphet** [comment]
Good catch on this one @loganbraywork.  

This was tricky to track down.  The problem was caused by the interaction between the `maxWidth` setting of the panel and the positioning of an invisible absolute value representation on the number line that is created but not used when a point is at position 0.  I have changed the code to set the position of this line even when it is invisible, and that seems to have solved the problem.
